network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -945,7 +945,6 @@ def get_banner(self):
             classname="collapsible collapsed",
         ),
     ]
-
     translatable_fields = [
```

Comment:
Hi @dlopezvsr! It looks like a rogue line may have been deleted 😅 Can we please reinsert it before merging? 

Thanks!